data/kramdown-rfc2629.erb

```diff
@@ -12,14 +12,25 @@
 <% end -%>
 ]>
 
-<% ps.arr("pi", false) do |pi, val| -%>
-<?rfc <%=pi%>="<%= {true => "yes", false => "no", nil => "yes"}[val] || val %>"?>
+<%
+  rfcattrs = ps.attrs("ipr", "docName=docname", "category=cat",
+                      "consensus", "submissionType=submissiontype", "xml:lang=lang",
+                      "number", "obsoletes", "updates", "seriesNo=seriesno")
+  pis = KramdownRFC::ParameterSet.new({})
+  ps.arr("pi", false) do |pi, val|
+    pis[pi] = {nil => "true", false => "false", "true" => "true"}[val] || val
+  end
```

Comment:
I tried this also, but it looked much worse:

```ruby
pis = KramdownRFC::ParameterSet.new(Hash[ps.arr("pi", false).map {|pi, val|
  ...
}])
```

Reply:
Well, you could build the hash first and then call ParameterSet.new on that.